virttest/virsh.py

```diff
@@ -4857,3 +4857,37 @@ def guestinfo(name, options="", **dargs):
     :return: CmdResult object.
     """
     return command("guestinfo %s %s" % (name, options), **dargs)
+
+
+def get_user_sshkeys(name, user, **dargs):
```

Comment:
Those code are already covered in https://github.com/avocado-framework/avocado-vt/pull/3058.

Reply:
@updated